src/invite/Invite.hooks.tsx

```diff
@@ -0,0 +1,36 @@
+import { isNil } from 'lodash'
```

Comment:
nit: Do we really need this? Is it better than `x != null`?

Reply:
Need? No, not at all. But I find `isNil` to be more expressive than `!=` with no cost. I have similar opinions for many lodash methods. If you have strong opinions I'm happy to have a conversation.